table-api/src/main/java/io/deephaven/api/updateby/spec/RollingFormulaSpec.java

```diff
@@ -87,11 +153,4 @@ final void checkFormula() {
             throw new IllegalArgumentException("formula must not be empty");
         }
     }
-
-    @Value.Check
-    final void checkParamToken() {
```

Comment:
I wonder if we could do a `Selectable.parse` up front to determine whether the output column name is specified when param token is missing. Maybe redundant, but that's sort of in the spirit of our immutables conventions. Could also conceivably make that `selectable()` a lazily-computed, cached field.